doc/build-osx.md

```diff
@@ -19,7 +19,7 @@ Then install [Homebrew](https://brew.sh).
 
 ## Dependencies
 ```shell
-brew install automake berkeley-db4 libtool boost miniupnpc pkg-config python qt libevent qrencode sqlite
+brew install automake libtool boost miniupnpc pkg-config python qt libevent qrencode
 ```
 
 If you run into issues, check [Homebrew's troubleshooting page](https://docs.brew.sh/Troubleshooting).
```

Comment:
It might be helpful to add a general note here that wallet support requires one or both of the dependencies in the sections below.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/20527#issuecomment-738075686).